### ISSUE TITLE
Dashboard search: topnav enabled, the keyboard shortcuts 'f' and 's o' do not work

### ISSUE THREAD

**Issue by krzysion** [grafana/grafana#65476]
After enabling the topnav feature in version 9.5.0-107958pre, the keyboard shortcuts 'f' and 's o' do not work, and it is not possible to open the dashboard search window.

**Environment**:
- Grafana version: v9.5.0-107958pre
- Data source type & version: Prometheus
- OS Grafana is installed on: Ubuntu
- User OS & Browser: Chrome
- Grafana plugins:
- Others:

**Comment by usmangt** [comment]
Hi @krzysion,

Thanks for opening this issue.

Based on your steps of reproduction, I was also able to reproduce it on my test machine.

I reproduced it using grafana-9.5.0~108290pre-1

![image](https://user-images.githubusercontent.com/69509548/228808757-6ab55639-0159-4bf8-9e86-95545e8fab4d.png)

I followed the help guide which says to use `s o` for searching the dashboard but does not work.

It does work for e.g. `g p` or `g e`

So seems like a bug

I will forward it to our Engineering team :+1:
